profile/arc_cwl_ro_crate.md

```diff
@@ -1,23 +1,339 @@
+
+### CWL Workflow Profile
+
+The CWL Workflow Profile extends the [Bioschemas ComputationalWorkflow Profile](https://bioschemas.org/profiles/ComputationalWorkflow/1.0-RELEASE#nav-description). It adds additional properties to describe the workflow in more detail using the [LabProcess](https://bioschemas.org/types/LabProcess/0.1-DRAFT).
+An example of the original profile can be found [here](https://www.researchobject.org/ro-crate/specification/1.1/workflows.html#complete-workflow-example)
+When compared to processes in a laboratory, a workflow is highly similar to a protocol. Protocols can be described using [PropertyValue](https://schema.org/PropertyValue). Workflow complexity can vary. Workflows executing several tools in succession are common and require more complex annotation. This can be achieved by using lists of property values.
+
+### CWL Workflow Run Profile
+
+The CWL Workflow Run Profile extends the [Workflow Run Crate](https://www.researchobject.org/workflow-run-crate/profiles/workflow_run_crate/). It adds additional properties to describe the parameters used in the workflow execution in more detail using the [LabProcess](https://bioschemas.org/types/LabProcess/0.1-DRAFT). When staying in the laboratory context, runs can be compared to performing the steps of a protocol. The steps involved in the execution of a protocoll can be described by a LabProcess. It may also contain information about inputs and outputs of the specific step.
 ```mermaid
 flowchart TD
-
+        A["File\nSoftwareSourceCode\nComputationalWorkflow"] -- "input\noutput" --> B["FormalParameter"]
+        A -- "instrument" --> C["CreateAction"]
+        C -- "agent" --> D["Person or Organization"]
+        B -- "exampleOfWork" --> E["File or Property Value"]
+        C -- "object result" --> E
+        C -- "about" --> H["Lab Process"]
 ```
+When adding the workflow run execution context, the "hasPart" field contains all files that are part of the invoced workflow. The "inputs" and "outputs" of the "ComputationalWorkflow" 
+MAY point to the "objects" and "results" of "CreateAction" via "workExample", while the latter point to the former via "exampleOfWork".
 
 ## Requirements
 
+### CWL Workflow Profile
+
+The requirements of this profile are those of [Bioschemas ComputationalWorkflow Profile](https://bioschemas.org/profiles/ComputationalWorkflow/1.0-RELEASE#nav-description) 
+plus the ones listed below.
+
+#### ComputationalWorkflow
 | Property | Required | Expected Type | Description |
 |----------|----------|---------------|-------------|
-| | | | |
+|about|SHOULD|[schema.org/PropertyValue](https://schema.org/PropertyValue)|The computational processes encoded in this workflow|
```

Comment:
I suggest to drop this as well, like for the Run, and make the Workflow of double type `ComputationalWorkflow` and `LabProtocol`.